src/javascript/DesignSystem/MultipleInput/components/Control.jsx

```diff
@@ -1,47 +1,75 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import {Input} from '@jahia/design-system-kit';
-import {InputComponent} from './InputComponent';
 import {withStyles} from '@material-ui/core';
 
 const styles = theme => ({
     input: {
-        paddingLeft: 0
+        paddingLeft: 0,
+        borderRadius: '1px',
+        background: theme.palette.ui.epsilon,
+        border: `1px solid ${theme.palette.ui.omega}`,
+        boxSizing: 'border-box',
+        '&:hover': {
+            border: `1px solid ${theme.palette.ui.zeta}`
+        },
+        fontSize: theme.typography.iota.fontSize,
+        transitionDuration: '.3s',
+        padding: '3px 0px 3px 12px',
+        display: 'flex',
+        minHeight: '44px'
+    },
+    focused: {
+        border: `1px solid ${theme.palette.brand.alpha}`
+    },
+    disabled: {
+        background: theme.palette.ui.epsilon,
+        border: `1px solid ${theme.palette.ui.zeta}`,
+        color: theme.palette.font.gamma
     },
     readOnly: {
         background: theme.palette.ui.alpha,
         border: `1px solid ${theme.palette.ui.alpha}`
     }
 });
 
-export const ControlCmp = React.forwardRef(({classes, children, readOnly, innerProps}, ref) => {
+export const ControlCmp = ({classes, innerRef, children, innerProps, isDisabled, isFocused, menuIsOpen, selectProps}) => {
+    const isReadOnly = selectProps.isReadOnly;
     return (
-        <Input
-            ref={ref}
-            fullWidth
-            inputComponent={InputComponent}
-            className={`${classes.input}
-                        ${readOnly ? classes.readOnly : ''}`
-            }
-            inputProps={{
-                inputRef: ref,
-                children: children,
-                ...innerProps
-            }}
-        />
+        <div
```

Comment:
div, encore! okay :3

Reply:
Control component is a wrapper for many components in react-select. The real input is in children ;)